### ISSUE TITLE
see file size in file browser 

### ISSUE THREAD

**Issue by ggrrll** [jupyter/notebook#3521]
Hi, 

I was wondering if there is a way (or if it's a planned feature ) to see the files size in the file browser, next to the modification time, for instance.

Thanks.

**Comment by takluyver** [comment]
I don't think it's on the roadmap, but I don't see a problem with adding it, if someone wants to work on it.

The frontend and the API should treat it as optional information, because some storage backends may not have an easy way to calculate a size.

**Comment by ckilcrease** [comment]
Sounds like a cool feature - could @ashleytqy, @rosaswaby, @danagilliann and I work on this together?

**Comment by takluyver** [comment]
Absolutely! In general, if no-one has indicated that they're working on an issue, there's no need to ask permission to tackle it. :-)

Can you see what bits of code you'll need to look at?

**Comment by takluyver** [comment]
Yep, that looks right. There are three kinds of model that the server can send to the frontend, for directories, notebooks, and files (i.e. files other than notebooks). It probably makes sense for notebooks and files to have a size field.
